From df761865ddaf3e110ca6dbfa7e34d43b802d1ea6 Mon Sep 17 00:00:00 2001
From: Philippe Schenker <philippe.schenker@toradex.com>
Date: Fri, 26 Aug 2022 15:03:42 +0200
Subject: [PATCH 194/233] [FIO fromlist] board: verdin-imx8mm: add call to
 ft_common_board_setup

With this call the following attributes get set to the device-tree
and are then accessible from linux in /proc/device-tree/

serial-number: The serial number that is stored in config-block
toradex,board-rev: The version of the module (e.g. V1.1A)
toradex,product-id: The SKU number of the module running

Fixes: commit 14d5aeff776b ("board: toradex: Add Verdin iMX8M Mini support")
Signed-off-by: Philippe Schenker <philippe.schenker@toradex.com>
Upstream-Status: Submitted [https://lore.kernel.org/u-boot/20220829175953.249441-1-dev@pschenker.ch/]
(cherry picked from commit 21c206960bd0b51afae8d8e21244698e099a0bfc)
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 board/toradex/verdin-imx8mm/verdin-imx8mm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/toradex/verdin-imx8mm/verdin-imx8mm.c b/board/toradex/verdin-imx8mm/verdin-imx8mm.c
index 7597cd81f94..ffcb04fc31e 100644
--- a/board/toradex/verdin-imx8mm/verdin-imx8mm.c
+++ b/board/toradex/verdin-imx8mm/verdin-imx8mm.c
@@ -128,6 +128,6 @@ int board_phys_sdram_size(phys_size_t *size)
 #if defined(CONFIG_OF_LIBFDT) && defined(CONFIG_OF_BOARD_SETUP)
 int ft_board_setup(void *blob, struct bd_info *bd)
 {
-	return 0;
+	return ft_common_board_setup(blob, bd);
 }
 #endif
-- 
2.25.1

